assets/bootstrap/js/app.js

```diff
@@ -0,0 +1,77 @@
+const cardSection = document.querySelector('[data-card-section]');
+
+const app = [{
+  cardClass: 'card',
+  image: 'assets/img/thumbnails/gallery_01_10_thumb.jpg',
+  alt: "Alex's car",
+  title: 'Alex',
+  subtitle: '2018 WRX',
+  text: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
+  cardClass1: 'card',
+  image1: 'assets/img/thumbnails/gallery_02_02_thumb.jpg',
+  alt1: "John's car",
+  title1: 'John',
+  subtitle1: '2018 WRX',
+  text1: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
+},
+{
+  cardClass: 'card',
+  image: 'assets/img/thumbnails/gallery_01_13_thumb.jpg',
+  alt: "Patrick's car",
+  title: 'Patrick',
+  subtitle: '2017 WRX STI',
+  text: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
+  cardClass1: 'card',
+  image1: 'assets/img/thumbnails/gallery_01_10_thumb.jpg',
+  alt1: "Alex's car",
+  title1: 'Alex',
+  subtitle1: '2018 WRX',
+  text1: 'Nullam id dolor id nibh ultricies vehicula ut id elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus.',
+},
```

Comment:
- [ ] Good job creating this section by JS, one more thing to make it complete is to fix the mobile design to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/164464447-a4ba9edd-7347-4431-becd-3988224627db.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/164464538-82cd0ea3-12de-4aaf-8256-eea4dbbbe985.png)

Reply:
- Fixed section to design specification
<img width="620" alt="Screenshot 2022-04-21 at 17 33 34" src="https://user-images.githubusercontent.com/40231363/164481690-ed693452-75c8-44dd-b775-11438aad24c7.png">